tools/bin/commands/util/operator_funcs

```diff
@@ -6,6 +6,23 @@ GO_BUILDER_IMAGE="syndesis/godev:1.10"
 # Name of the operator image
 OPERATOR_IMAGE="syndesis/syndesis-operator"
 
+# globs to test
+TEST_TARGETS="./cmd/... ./pkg/..."
+
+# $GOPATH volume to share between docker runs
+DOCKER_GOPATH_VOLUME=$(mktemp -d ${TMPDIR:-/tmp}/gopath.XXXXX)
```

Comment:
Shouldn't that be set only when no `--local` is given ? Also I would add them already here to $CLEANUP_DIRS so is clear where it belongs together.